Client/Frontend/Widgets/SiteTableViewHeader.swift

```diff
@@ -36,7 +31,7 @@ class SiteTableViewHeader: UITableViewHeaderFooterView, ThemeApplicable, Reusabl
     }
 
     private let collapsibleImageView: UIImageView = .build { imageView in
-        imageView.image = ExpandButtonState.down.image?.tinted(withColor: UIColor.Photon.Blue50)
+        imageView.image = ExpandButtonState.down.image
     }
```

Comment:
nit: is it worth it to set the image here without tint color, since it will be overriden anyway in applyTheme?